### ISSUE TITLE
Improve UX of configure command

### ISSUE THREAD

**Issue by benarmston** [alces-flight/flight-asset-cli#12]
The UX of the configuration command could do with some love.  The goal of the changes here are to  make the configuration command behave in a more similar manner to the `aws configure` command.

The configuration is to be split into two categories: those managed by `flight asset configure` and those not managed by `flight asset configure`.  These two groups of values are to be stored in two separate configuration files.  Obviously, one of these is tool-managed and the other is not.

The values managed by the new configuration command are the component id and the API token.

The existing `configure` command should be replaced with one that takes no options and no arguments.  That is `flight asset configure`.  This should prompt the user for the token and component ID.

If present, the existing value for the configuration should be shown and truncated to 8 characters.  If the value has not yet been shown, the string `None` should be used.

If the user does not enter a new value, or enters a new value consisting of only whitespace, the existing values should continue to be used.

Examples:

Initial configuration:

```
$ flight asset configure
Flight Center API token [None]: my-long-token
Component identifier [None]: 3
```

Update token:

```
$ flight asset configure
Flight Center API token [my-long-t...]: my-new-long-token
Component identifier [4]: <enter -- i.e. accepts default, leaves unchanged>
```

Other configuration values can be managed by the packaging system and/or advanced users by editing the config file.  This, more static, config file should be at: `${flight_asset_ROOT}/etc/config.yml`.

**Comment by WilliamMcCumstie** [comment]
WIP

**Comment by WilliamMcCumstie** [comment]
Was mostly implemented in: a6c7c80f83eac74bfe94c88deaa9d816bb17cfe5

There have been a couple of minor tweaks since then
